.github/workflows/step_tests.yml

```diff
@@ -0,0 +1,57 @@
+name: PennyLane-Orquestra component tests
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+
+env:
+  COVERAGE_FLAGS: "--cov=steps --cov-report=term-missing --cov-report=xml --no-flaky-report -p no:warnings --tb=native"
+
+
+jobs:
+  core-tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        config:
+          - {python-version: 3.7}
+          - {python-version: 3.8}
```

Comment:
Is there a reason for 3.6 not being part of these tests (as they are in the other test file)?

Reply:
Yes, Orquestra doesn't support Python version <3.7.